backend/coins/btc/addresses/address.go

```diff
@@ -53,71 +53,81 @@ type AccountAddress struct {
 
 // NewAccountAddress creates a new account address.
 func NewAccountAddress(
+	singleAddress btcutil.Address,
```

Comment:
Yes, but account `accountConfiguration` has the address as string, not `btcutil.Address` so I'd have to decode it here, instead of in account.go. 
Thought we previously talked about doing it this way, no problem changing it though if it makes more sense to decode it here instead